### ISSUE TITLE
[BUG] Script closes immediately - API response KeyError

### ISSUE THREAD

**Issue by Primus27** [Primus27/R6-Profile-Switcher#1]
The script runs and closes immediately after the API call.

**Output:**
```
R6 Profile Switcher (v3.1.0)
 \- Developed by Primus27 (github.com/primus27)

Traceback (most recent call last):
  File "switch_v3.py", line 278, in <module>
    acc_resolve_list = [resolve_uplay_info(player_id) for player_id
  File "switch_v3.py", line 278, in <listcomp>
    acc_resolve_list = [resolve_uplay_info(player_id) for player_id
  File "switch_v3.py", line 142, in resolve_uplay_info
    player_name = json_info["player"]["p_name"]
KeyError: 'player'
```

**Comment by Primus27** [comment]
This happens because the API response (a code 200) returns missing information. For example, the API returns information on why it can't process the request (such as missing API token that was not previously required).

Example response from R6Tabs API:
`{"authorized":false,"API":{"code":"NO_KEY"},"found":false,"status":403,"error":"Not authorized!","message":"You need an API key.","contact":"https:\/\/tabstats.com\/contact"}`

At the moment, I'm waiting on information from the R6Tabs team.

**Comment by Primus27** [comment]
Solved in version 3.1.1.
